package.json

```diff
@@ -44,26 +44,26 @@
     "@endpass/class": "^0.20.1",
```

Comment:
@dudiq don't work for me.
1. got documents list
2. logout from vault
3. without page refresh, try to sign in in demo
4. got same documents, without login

Reply:
try again, connect and auth updated